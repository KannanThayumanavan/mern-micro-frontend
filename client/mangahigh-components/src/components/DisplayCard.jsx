import React from 'react';
import PropTypes from 'prop-types';

const DisplayCard = ({ highlight, summary }) => (	
	<div className="card border-success mb-3">
	  <div className="card-header">{summary}</div>
	  <div className="card-body text-success">
	    <h5 className="card-title h1">{highlight}</h5>		    
	  </div>
	</div>
);

DisplayCard.propTypes = {
	highlight: PropTypes.string.isRequired, 
	summary: PropTypes.string.isRequired,
};
export default DisplayCard;