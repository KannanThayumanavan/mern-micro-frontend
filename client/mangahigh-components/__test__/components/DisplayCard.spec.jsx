import React from 'react';
import { shallow } from 'enzyme';
import DisplayCard from '../../src/components/DisplayCard';

describe('Test DisplayCard Component', () => {
	it('with required props', () => {
		const renderedModule = shallow(
      <DisplayCard
      highlight='16'
      summary='times attempted'
    />);
    expect(renderedModule).toMatchSnapshot();
	});
});