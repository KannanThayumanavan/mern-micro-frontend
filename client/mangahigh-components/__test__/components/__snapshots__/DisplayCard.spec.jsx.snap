// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DisplayCard Component with required props 1`] = `
<div
  className="card border-success mb-3"
>
  <div
    className="card-header"
  >
    times attempted
  </div>
  <div
    className="card-body text-success"
  >
    <h5
      className="card-title h1"
    >
      16
    </h5>
  </div>
</div>
`;
